File: EasyTalentos.Web/ClientApp/src/app/models/Preferencia.ts
import { Candidato } from "./Candidato";

export class Preferencia {
  id: number;
  manha: boolean = false;
  tarde: boolean = false;
  noite: boolean = false;
  madrugada: boolean = false;
  comercial: boolean = false;
  candidatoId: number;
  candidato: Candidato;
}

File: EasyTalentos.Dominio/Entity/Preferencia.cs
using System.Text.Json.Serialization;

namespace EasyTalentos.Dominio.Entity
{
    public class Preferencia
    {
        public int Id { get; set; }

        public bool Manha { get; set; }
        public bool Tarde { get; set; }
        public bool Noite { get; set; }
        public bool Madrugada { get; set; }
        public bool Comercial { get; set; }

        public int CandidatoId { get; set; }

        [JsonIgnore]
        public virtual Candidato Candidato { get; set; }
    }
}
